beagle/src/main/kotlin/br/com/zup/beagle/android/view/BeagleFragment.kt

```diff
@@ -17,19 +17,29 @@
 package br.com.zup.beagle.android.view
 
 import android.os.Bundle
+import android.os.Handler
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.FrameLayout
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleObserver
```

Comment:
<table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-android/blob/e6e212aeddcd8b91f8c031a9c9d31a35779e51a8/beagle/src/main/kotlin/br/com/zup/beagle/android/view/BeagleFragment.kt#L27"></span>The import 'androidx.lifecycle.LifecycleObserver' is unused.</td>
    </tr>
  </tbody>
</table>